src/mscorlib/shared/Interop/Unix/Interop.Libraries.cs

```diff
@@ -8,5 +8,6 @@ internal static partial class Libraries
     {
         internal const string GlobalizationInterop = "System.Globalization.Native";
         internal const string SystemNative = "System.Native";
+        internal const string LibCoreCLR = "libcoreclr";
```

Comment:
@yizhang82 Should the BSTR alloc/free be abstracted behind PInvokeMarshal - similar to what is done for regular memory allocations?